ElectronClient/app.js

```diff
@@ -1166,7 +1169,7 @@ class Application extends BaseApplication {
 		this.lastMenuScreen_ = screen;
 	}
 
-	async updateMenuItemStates(state = null) {
+	async updateMenuItemStates(layout, state = null) {
```

Comment:
i also renamed the `viewer` variable to `layout` because it connotes the meaning of the value it stores.